prep-javascript-variables/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>JavaScript Variables</title>
+  </head>
+  <body>
+    <script>
+      var fullName = 'Lisa Le';
+      console.log('value of fullName: ' + fullName);
+
+      var numberOfPets = 1;
+      console.log('value of numberOfPets: ' + numberOfPets);
+
+      var likesVegetable = 'false';
+      console.log('value of likesVegetables: ' + likesVegetable);
```

Comment:
For Boolean values, you should leave off the quotation marks so that JavaScript doesn't interpret them as strings.